domain/schema/model/sql/0022-resource.sql

```diff
@@ -100,16 +112,12 @@ CREATE TABLE resource_supplied_by (
 CREATE UNIQUE INDEX idx_resource_supplied_by
 ON resource_supplied_by (name);
 
-CREATE TABLE application_resource (
+-- This is a resource used by a kubernetes application.
+CREATE TABLE kubernetes_application_resource (
```

Comment:
I'm not sure I understand the purpose of this table. If a resource is used by a Kubernetes application surely the fact that it is a kubernetes app is contained in the application table somewhere. If this table is instead saying that the resource is an OCI image resource then this superseded by the `resource_store` table, which will reference `resource_container_image_metadata_store` (though this cannot be enforced via foreign key)